MiniApp/Classes/MiniAppSdkConfig.swift

```diff
@@ -11,13 +11,21 @@ public class MiniAppSdkConfig {
             }
         }
     }
+    @available(*, deprecated)
```

Comment:
Can you add a deprecation message that they should use `rasProjectId` instead?

Reply:
Done